README.md

```diff
@@ -295,5 +298,32 @@ If an authorization error similar to the one below appears:
 ```
 Ensure that you consented to the following API scopes as a user with the proper [permissions to consent](#permissions) and have enabled the required [APIs and Services](#create-an-oauth-credential).
 
+### Unable to view HTML report due to environment limitations 
+
+If you are unable to view the HTML report in a browser window, the results of the complicance scan can be viewed in their raw JSON format. 
```

Comment:
On M365 they have been using "conformance" instead of "compliance."

```suggestion
If you are unable to view the HTML report in a browser window, the results of the conformance scan can be viewed in their raw JSON format. 
```

Reply:
Changed all instances of 'compliance' to 'conformance'